.gitignore

```diff
@@ -82,7 +82,7 @@ __pycache__/
 /src/libstd_unicode/SpecialCasing.txt
 /src/libstd_unicode/UnicodeData.txt
 /stage[0-9]+/
-/target
+target/
```

Comment:
Ah unfortunately this has the unintended side effect of ignoring `src/librustc_back/target`, so perhaps these changes could be left out for now?